utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
```

Comment:
This file needs a license header. Take a look at the other Python files in the repository for examples.